pos_order_mgmt/static/src/js/widgets.js

```diff
@@ -324,12 +335,14 @@ odoo.define('pos_order_mgmt.widgets', function (require) {
                 // Invert line quantities
                 qty *= -1;
             }
-
             return {
                 price: line.price_unit,
                 quantity: qty,
                 discount: line.discount,
                 merge: false,
+                extras: {
+                    return_lot_no: line.lot_name,
```

Comment:
And if you renamed the field before, here it would be

```suggestion
                    return_pack_lot_names: line.pack_lot_names,
```